website/docs/create-new-theme.md

```diff
@@ -0,0 +1,47 @@
+---
+id: create-new-theme
+title: Create new WordPress theme
+sidebar_label: Create New Theme
+---
+
+## Requirements
+
+1. [Node.js](https://nodejs.org/en/)
+2. [Composer](https://getcomposer.org/)
+3. [WP-CLI](https://wp-cli.org/)
+4. [Git](https://git-scm.com/)
+
+Please make sure that you have all this packages installed and
+ready to use on your system. To make sure that they work run this commands:
+```bash
+node -v
+composer -v
+wp --info
+git --version
+```
+
+If it doesn't return any errors you are good to go.
+
+## Quick start
+Let's create a new theme!
+
+Navigate to your WordPress theme folder and run the following command:
+
+```bash
+npx create-wp-project
+```
+
+Script will prompt you for theme name and local development url (used for BrowserSync) and install a new theme:
```

Comment:
The script will prompt you for a theme name and local development URL (used for BrowserSync). After that your new theme will be installed: